applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -466,28 +486,36 @@ static int do_secure_socketopt_set_int(int option, int value)
 
 static int do_secure_socketopt_get(int option)
 {
+	int ret = -ENOTSUP;
+	int value;
+	socklen_t len = sizeof(int);
+
 	switch (option) {
 	case TLS_CIPHERSUITE_USED: /* MFW >= 2.0.0 */
+		ret = getsockopt(sock.fd, SOL_TLS, option, &value, &len);
+		if (ret) {
+			LOG_ERR("getsockopt(%d) error: %d", option, -errno);
+		} else {
+			rsp_send("\r\n#XSSOCKETOPT: 0x%x\r\n", value);
+		}
```

Comment:
does this need to be a separate `case` block?
We do the same operation as in all other `case`.

Reply:
Cipher suites are usually given in hex format. For example in here: https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml.

I agree that it would be cleaner in decimal format, but in this case formatting has a benefit. That said, if you think that it should be in decimal format, I can change it.

Reply:
ok as-is. Had not noticed the print format is different.